icu4c/source/common/uloc_tag.cpp.orig

```diff
@@ -0,0 +1,2534 @@
+// © 2016 and later: Unicode, Inc. and others.
```

Comment:
Should this file be included?

Reply:
oops.  No. I'll delete them, too. Hmm.. I don't know what I was doing.. 

Thank you for catching it.